mate-panel/libmate-panel-applet-private/panel-applet-container.c

```diff
@@ -499,27 +495,25 @@ mate_panel_applet_container_get_applet (MatePanelAppletContainer *container,
 				   GAsyncReadyCallback   callback,
 				   gpointer              user_data)
 {
-	GSimpleAsyncResult *result;
+	GTask              *task;
 	AppletFactoryData  *data;
 	gint                screen_number;
 	gchar              *bus_name;
 	gchar              *factory_id;
 	gchar              *applet_id;
 
-	result = g_simple_async_result_new (G_OBJECT (container),
-					    callback,
-					    user_data,
-					    mate_panel_applet_container_get_applet);
-
+	task = g_task_new (G_OBJECT (container),
+                                    cancellable,
+                                    callback,
+                                    user_data);
```

Comment:
Line 506-508 should be aligned with `G_OBJECT` of line 505
And it looks worse here in browser because you didn't use tabs like all over this file.
Should be...
```
-	result = g_simple_async_result_new (G_OBJECT (container),
-					    callback,
-					    user_data,
-					    mate_panel_applet_container_get_applet);
-
+	task = g_task_new (G_OBJECT (container),
+			   cancellable,
+			   callback,
+			   user_data);
```